pkg/controllers/nodeclaim/disruption/controller.go

```diff
@@ -134,6 +135,14 @@ func (c *Controller) Builder(_ context.Context, m manager.Manager) operatorcontr
 								return true
 							}
 						}
+						for _, cond := range []apis.ConditionType{v1beta1.Expired, v1beta1.Empty, v1beta1.Drifted} {
```

Comment:
Do we just outright remove the event filtering logic here and just requeue across all updates? I'm trying to think of a case where that wouldn't make sense as long as we are ensuring that our updates don't cause increased reconciles